packages/opentelemetry/src/propagator.ts

```diff
@@ -338,3 +356,24 @@ export function getSamplingDecision(spanContext: SpanContext): boolean | undefin
 
   return undefined;
 }
+
+/** Try to get the existing baggage header so we can merge this in. */
+function getExistingBaggage(carrier: unknown): string | undefined {
+  try {
+    const baggage = (carrier as Record<string, string | string[]>)[SENTRY_BAGGAGE_HEADER];
+    return Array.isArray(baggage) ? baggage.join(',') : baggage;
+  } catch {
+    return undefined;
+  }
+}
+
+/**
+ * Will parse a baggage header into an array of tuples,
+ *  where the first item is the baggage name, the second the baggage value.
+ */
+function parseBaggageHeaderString(baggageHeader: string): [string, string][] {
+  return baggageHeader.split(',').map(baggageEntry => {
+    const [key, value] = baggageEntry.split('=');
+    return [decodeURIComponent(key.trim()), decodeURIComponent(value.trim())];
+  });
+}
```

Comment:
l: Is there a reason we need to parse this into an array of tuples? just wondering if it makes sense to unify things here with `baggageHeaderToObject` from `@sentry/utils`?

Feel free to disregard if N/A :)

Reply:
Yeah, that makes sense!